cmd/echofid/cmd/command.go

```diff
@@ -67,17 +71,30 @@ func initRootCmd(rootCmd *cobra.Command,
 
 	server.AddCommands(rootCmd, app.DefaultNodeHome, ac.newApp, ac.appExport, addModuleInitFlags)
 
+	evmserver.AddCommands(
+		rootCmd,
+		evmserver.NewDefaultStartOptions(ac.newApp, app.DefaultNodeHome),
+		ac.appExport,
+		addModuleInitFlags,
+	)
+
 	// add keybase, auxiliary RPC, query, and tx child commands
 	rootCmd.AddCommand(
 		server.StatusCommand(),
 		genesisCommand(txConfig, basicManager),
 		queryCommand(),
 		txCommand(basicManager),
-		keys.Commands(),
+		evmcmd.KeyCommands(app.DefaultNodeHome, false),
```

Comment:
Let careful with `false` or `true` here it decides the default keyring algo. 
`false` : default cosmos key
`true` : default eth_secp256k1 key